ui/src/components/admin/users/AdminUser.js

```diff
@@ -444,9 +414,12 @@ export default function AdminUser(props) {
       ...data,
     });
 
-    if (res.status !== 200) {
+    if (res.status !== 200 || res.data.id == null || res.data.id === '') {
```

Comment:
why for the comparison of res.data.id, you used 2 different equal signs: '==' and '==='?  just curious. i dont think it affects anything

Reply:
`== null` checks if `id` is `null` or `undefined`, `=== '"` checks if `id` is string type and is not empty string